Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2018"
 cli = { package = "polkadot-cli", path = "cli" }
 futures = "0.3.4"
 service = { package = "polkadot-service", path = "service" }
-parity-util-mem = { version = "0.5.1", default-features = false, features = ["jemalloc-global"] }
+parity-util-mem = { version = "0.6.0", default-features = false, features = ["jemalloc-global"] }
```

Comment:
cc @cheme, it looks like 0.6 version was used indirectly (via kvdb) w/o jemalloc-global feature, so it expected `malloc` to be the global alloc https://github.com/paritytech/parity-common/blob/939151e23b132110628739e8458e6cece1f1c8d0/parity-util-mem/src/allocators.rs#L99-L106, and using `parity-util-mem` here to enforce `jemalloc` as the global allocator seems flaky

Reply:
I would suggest at least changing the version to `* ` and making sure only one version of `parity-util-mem` is used.